ci/tests/vdisk/vdisk_test.py

```diff
@@ -89,3 +85,35 @@ def _get_scrubber_log_size():
                                    loop_device=loop)
 
         assert post_scrubber_logsize > pre_scrubber_logsize, "Scrubber actions where not logged!"
+
+    @staticmethod
+    def ovs_3756_metadata_size_test():
+        """
+        Validate get/set metadata cache size for a vdisk
+        """
+
+        DEFAULT_METADATA_CACHE_SIZE = 8192 * 256 * 24
```

Comment:
This value is never used + upper case should, according to Pycharm, only be used outside methods, but in the class itself